pkgs/servers/ankisyncd/default.nix

```diff
@@ -1,62 +1,55 @@
 { lib
-, fetchFromGitHub
 , python3
-, anki
+, pkgs
```

Comment:
Is there any reason to use `pkgs`?

Reply:
Why was this marked as resolved? What's the reason?